lib/runtime-c-api/wasmer.h

```diff
@@ -0,0 +1,249 @@
+#ifndef WASMER_H
+#define WASMER_H
+
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
```

Comment:
Maybe we should add a:

```cpp
#ifdef __cplusplus
extern "C" {
#endif
```

And a 
```cpp
#ifdef __cplusplus
}  // extern "C"
#endif
```
at the end

Reply:
@syrusakbary There's another file `wasmer.hh` that is generated for `cpp`.

Reply:
Thanks for the clarification! Resolving my comment :)